README.md

```diff
@@ -294,8 +294,26 @@ Run secrets scanning on all files in MyDirectory. Skip CKV_SECRET_6 check on jso
 checkov -d /MyDirectory --framework secrets --bc-api-key ... --skip-check CKV_SECRET_6:.*skip_test.*json$
 ```
 
+Supply a path to config file (JSON format) that contains mapping between resource and specific attributes it might contain.
+Checkov will use this config to mask (obfuscate) the resource attributes values.
+For example:
+ - config file that contains (detailed below) will cause the azurerm_key_vault_secret.value to be masked.
+ ```sh
+ { "azurerm_key_vault_secret": ["value"]}
+```
+ - config file that contains (detailed below) will cause the {RESOURCE} that has value attribute to be masked.
+ ```sh
+ { "*": ["value"]}
+```
+Usage example:
+```sh
+checkov -d --framework terraform_plan --resource-attr-to-omit PATH_TO_FILE.json
+```
```

Comment:
I think the intention was to add the config params to the existing checkov config file, so we don't require users to create new files just for this.
So the existing config file could now be extended to look like this:
```
framework:
- all
secrets-scan-file-type: []
summary-position: top
mask:
- password_variable                # this is a univeral mask
- resource_type/attribute        # this is a specific resource attribute mask
```
Unless these specs were changed, the config structure should be changed so it doesnt require a new file